cloudinit/osys/windows/base.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2015 Cloudbase Solutions Srl
+# Copyright 2015 Canonical Ltd.
+# License TBD
+
+from cloudinit.osys import base
+from cloudinit.osys.windows import filesystem as filesystem_module
+from cloudinit.osys.windows import general as general_module
+from cloudinit.osys.windows import network as network_module
+from cloudinit.osys.windows import users as users_module
+
+
+__all__ = ('OSUtils', )
+
+
+class OSUtils(base.BaseOSUtils):
+    """The OS utils namespace for the Windows platform."""
+
+    name = "windows"
+
+    network = network_module.Network()
+    filesystem = filesystem_module.Filesystem()
+    users = users_module.Users()
+    general = general_module.General()
+    user_class = users_module.User
+    route_class = network_module.Route
+    interface_class = network_module.Interface
```

Comment:
At this point, I didn't feel that it was needed. On the other hand, route_class / user_class / interface_class are actually used for controlling various aspects of what they represent (they have some class methods for creating / deleting objects basically).